linux_setup.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# The user must define this path
```

Comment:
Please add an appropriate copyright and license header (see https://github.com/edgexfoundry/developer-scripts/blob/master/run-it.sh for an example)